camera_lucid.orogen

```diff
@@ -10,46 +10,38 @@ import_types_from "camera_lucidTypes.hpp"
 # library's pkg-config name) and then the header can be used. Following Rock
 # conventions, a common use-case would be:
 #
-# using_library "camera_lucid"
+using_library "camera_lucid_arenasdk"
+# import_types_from "camera_lucid_arenasdk"
 # import_types_from "camera_lucid/CustomType.hpp"
 
 # If this project uses data types that are defined in other oroGen projects,
 # these projects should be imported there as well.
 import_types_from "std"
-# import_types_from "base"
+import_types_from "base"
 
-# Declare a new task context (i.e., a component)
-#
-# The corresponding C++ class can be edited in tasks/Task.hpp and
-# tasks/Task.cpp, and will be put in the camera_lucid namespace.
 task_context "Task" do
-    # This is the default from now on, and should not be removed. Rock will
-    # transition to a setup where all components use a configuration step.
     needs_configuration
 
-    # A configuration property (here, a std::string). Its value can be retrieved
-    # in the C++ code with # _config_value.get() and _config_value.set(new_value).
-    property "config_value", "/std/string"
-
-    # An input port, i.e. an object from which the component gets data from
-    # other components' outputs
-    #
-    # Data can be retrieved using _input.read(value), which returns true if data
-    # was available, and false otherwise. _input.connected() returns if this
-    # input is connected to an output or not.
-    input_port "input", "/std/string"
-
-    # An output port, i.e. an object to which the component pushes data so that
-    # it is transmitted to other components' inputs
-    #
-    # Data can be written using _output.write(value). _output.connected() returns
-    # if this output is connected to an input or not.
-    output_port "output", "int"
-
-    # If you want that component's updateHook() to be executed when the "input"
-    # port gets data, uncomment this and comment the 'periodic' line
-    # port_driven "input"
-    # By default, the task will be periodic with a period of 0.1
+    property "ip", "/std/string"
+    property "frame_timeout", "/base/Time"
+    property "frame_rate", "double"
```

Comment:
From what I saw it was double. I thought it was odd too, but I followed what they used. This and enums I'll get all values tomorrow when I go to the office. The documentation lacks the data types I'll get them directly from the camera params.

Reply:
```
Acquisition Frame Rate                        RO     Beginner      intfIFloat          2.06
```

Reply:
That's a read-only property. Even assuming the read-write one is the same type, I am guessing it is some form of scaled integer from the `intfIFloat` type. Anything in the documentation about that ?

Reply:
It is read only but if you wanna change this there is the param :
```
Acquisition Frame Rate Enable                 RW     Expert        intfIBoolean        0
```
that makes the AcquisitionFrameRate writable. intfIFloat is `interface type float`. I didn't find anything explaining why they use it as double instead of int.

Reply:
Alright then ...